src/components/dialogs/line-attach-to-voltage-level-dialog.js

```diff
@@ -53,6 +53,7 @@ const getId = (e) => e?.id || (typeof e === 'string' ? e : '');
  * @param {Boolean} open Is the dialog open ?
  * @param {EventListener} onClose Event to close the dialog
  * @param fetchedLineOptions Promise handling list of network lines
+ * @param fetchedVoltageLevelOptions Promise handling list of network voltage levels
```

Comment:
Could we take the occasion to rename these Promises to names saying so, like voltageLevelsPromise ?

Reply:
In another PR